### DISCUSSION TITLE
set-associative cache ways

### DISCUSSION THREAD

**Discussion by cf-natali** [dendibakh/perf-book#24] [Improvements & Bugs]
https://github.com/dendibakh/perf-book/blob/696c52af486de7d9fb881a9be8312a86347453f7/chapters/3-CPU-Microarchitecture/3-5%20Memory%20Hierarchy.md?plain=1#L28

```
An intermediate option between the direct mapping and fully associative mapping is a set-associative mapping. In such a cache, the blocks are organized as sets, typically each set containing 2,4 or 8 blocks.
```

On Zen 4, the [L3 is 16-way](https://en.wikichip.org/wiki/amd/microarchitectures/zen_4#Data_and_Instruction_Caches), might be worth updating?

**Comment by dendibakh** [comment]
Yes, feel free to submit a PR.

**Comment by cf-natali** [comment]
https://github.com/dendibakh/perf-book/pull/25
